package.json

```diff
@@ -296,5 +297,19 @@
         "`//unpkg.com/route-mini-app@^5.0.0/page-${this.page}.mjs`"
       ]
     ]
-  }
+  },
+  "bundlesize": [
+    {
+      "path": "./core.min.mjs",
+      "maxSize": "101 kB"
+    },
+    {
+      "path": "./core.mjs",
+      "maxSize": "311 kB"
+    },
+    {
+      "path": "./dist/global/core.js",
+      "maxSize": "182 kB"
```

Comment:
All of these sizes are within 1 kB of the current sizes. Maybe these should allow for more growth? I personally think that erring on the side of being too restrictive is preferable because we might otherwise not notice these files growing a bit at a time if we have, say, 5 or 10 kB of headroom.

Reply:
If we're going to make file size a priority, which we haven't in the past, then having little to know headroom is a good way to go. Every file size increase should be approved.